alpha/apps/kaltura/lib/live/kBroadcastUrlManager.php

```diff
@@ -92,13 +92,19 @@ protected function getConfiguration ($dc = null)
 		$partner = PartnerPeer::retrieveByPK($this->partnerId);
 		return $partner->getLiveStreamBroadcastUrlConfigurations($dc);
 	}
+
+	private static function getLiveIdForHost(BaseEntry $entry)
+	{
+		$entryId = str_replace('1_', '', $entry->getId());
```

Comment:
can we have old entries with 0_ on this case?

Reply:
Yes - in the '0_' case the underscore will be replaced with '-'